kytos/core/interface.py

```diff
@@ -505,6 +768,18 @@ def as_json(self):
         """
         return json.dumps(self.as_dict())
 
+    def notify_link_available_tags(self, controller, src_func=None):
+        """Notify link available tags"""
+        name = "kytos/core.link_available_tags"
```

Comment:
`link_available_tags` was named when it belonged to link, now it's an interface being sent, the name should change to `kytos/core.interface_available_tags` or something similar.

Reply:
Changed named to `kytos/core.interface_tags`